llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -51,7 +51,12 @@ void ComputedShaderFlags::print(raw_ostream &OS) const {
   if (FlagVal == 0)
     return;
   OS << "; Note: shader requires additional functionality:\n";
-#define SHADER_FEATURE_FLAG(bit, FlagName, Str)                                \
+#define SHADER_FEATURE_FLAG(FeatureBit, DxilModuleNum, FlagName, Str)          \
+  if (FlagName)                                                                \
+    OS << ";       " Str "\n";
```

Comment:
i'm not sure what the llvm equivalent is but seems like we should be doing something like std::setw.

Reply:
Fixed with raw_ostream::indent.

Reply:
Fixed with raw_ostream::indent.